.changeset/lovely-foxes-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'ui': minor
+---
+
+fix MsgDeposit type interpolation
```

Comment:
It appears that the this changeset is identical to the "fix MsgDeposit interpolation issue" changeset. 
Can you clarify the intention or purpose of this?

Reply:
yeah it's a duplicate, will remove the redundant changeset, thanks!